docs/topics/mta-7-configuration-web-console-on-ocp-local.adoc

```diff
@@ -0,0 +1,58 @@
+:_content-type: CONCEPT
+[id="mta-7-configuration-web-console-on-ocp-local_{context}"]
+= Configuration of the {ProductName} Operator in a {rh-ocp-local} environment
+
+The following table shows the different configurations of {rh-ocp-local} tested and the results obtained:
+
+[width="100%",cols="25%,25%,25%,25%",options="header",]
+|===
+|Memory (GiB)
+|CPU
+|Disk sze (GiB)
+|Comments
+
+|15
+|4 (default)
+|200
+|MTA does not run analysis due to insufficient memory and CPU.
```

Comment:
I think this first row can be removed since the results are the same as the next one.